src/main/kotlin/dsm/wemeet/domain/friend/service/impl/QueryFriendServiceImpl.kt

```
@@ -0,0 +1,12 @@
+package dsm.wemeet.domain.friend.service.impl
+
+import dsm.wemeet.domain.friend.repository.FriendJpaRepository
+import dsm.wemeet.domain.friend.service.QueryFriendService
+import org.springframework.stereotype.Service
+
+@Service
+class QueryFriendServiceImpl(
+    private val friendJpaRepository: FriendJpaRepository
+) : QueryFriendService {
+
+}
```

detekt (1.23.7)

[warning] The class or object QueryFriendServiceImpl is empty.

(detekt.empty-blocks.EmptyClassBlock)